backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ToplevelPhases.kt

```diff
@@ -317,17 +317,23 @@ internal val linkerPhase = konanUnitPhase(
         description = "Linker"
 )
 
-internal val allLoweringsPhase = namedIrModulePhase(
-        name = "IrLowering",
-        description = "IR Lowering",
+internal val preInlinePhase = namedIrModulePhase(
+        name = "IrPreInlineLowering",
+        description = "IR lowerings before inlining",
```

Comment:
Did we have similar issues with other lowerings from the sequence below?

Reply:
Correct me if I wrong but the inline lowering is the only lowering which requires function bodies from other modules, other only require headers and for headers there are some maps in `context` to not be dependent on the order of modules.

Reply:
I mean that `lateinitPhase`, for example, is probably not very different from `sharedVariablesPhase`.
What happens if  `inline` function from other module has `lateinit var`?

Reply:
I don't get it. We run all lowerings that are run before the inline phase for entire IR, `lateinitPhase` included.

Reply:
I mean _before your fix_.

Reply:
Ah, I see, you're implying to try to find similar bugs and add tests on'em?